integration/spark/src/main/scala/org/apache/spark/sql/execution/command/mutation/merge/CarbonMergeDataSetUtil.scala

```diff
@@ -0,0 +1,206 @@
+                joinDataType)
+              // check here if present in map, if it is, compare and update min and amx
+              if (fileMinMaxMap.contains(uniqBlockPath)) {
+                val isMinLessThanMin =
+                  comparator.compare(fileMinMaxMap(uniqBlockPath)._1, minValue) > 0
+                val isMaxMoreThanMax =
+                  comparator.compare(maxValue, fileMinMaxMap(uniqBlockPath)._2) > 0
+                updateMapIfRequiredBasedOnMinMax(fileMinMaxMap,
+                  minValue,
+                  maxValue,
+                  uniqBlockPath,
+                  isMinLessThanMin,
+                  isMaxMoreThanMax)
+              } else {
+                fileMinMaxMap += (uniqBlockPath -> (minValue, maxValue))
+              }
+            } else {
+              if (fileMinMaxMap.contains(uniqBlockPath)) {
+                val isMinLessThanMin = ByteUtil.UnsafeComparer.INSTANCE
+                                         .compareTo(fileMinMaxMap(uniqBlockPath)._1
+                                           .asInstanceOf[String].getBytes(), minBytes) > 0
+                val isMaxMoreThanMax = ByteUtil.UnsafeComparer.INSTANCE
+                                         .compareTo(maxBytes, fileMinMaxMap(uniqBlockPath)._2
+                                           .asInstanceOf[String].getBytes()) > 0
+                updateMapIfRequiredBasedOnMinMax(fileMinMaxMap,
+                  new String(minBytes),
+                  new String(maxBytes),
+                  uniqBlockPath,
+                  isMinLessThanMin,
+                  isMaxMoreThanMax)
+              } else {
+                fileMinMaxMap += (uniqBlockPath -> (new String(minBytes), new String(maxBytes)))
+              }
+            }
+          } else {
+            val maxValue = DataTypeUtil.getMeasureObjectFromDataType(maxBytes, joinDataType)
+            val minValue = DataTypeUtil.getMeasureObjectFromDataType(minBytes, joinDataType)
+            if (fileMinMaxMap.contains(uniqBlockPath)) {
+              val isMinLessThanMin =
+                comparator.compare(fileMinMaxMap(uniqBlockPath)._1, minValue) > 0
+              val isMaxMoreThanMin =
+                comparator.compare(maxValue, fileMinMaxMap(uniqBlockPath)._2) > 0
+              updateMapIfRequiredBasedOnMinMax(fileMinMaxMap,
+                minValue,
+                maxValue,
+                uniqBlockPath,
+                isMinLessThanMin,
+                isMaxMoreThanMin)
+            } else {
+              fileMinMaxMap += (uniqBlockPath -> (minValue, maxValue))
+            }
+          }
+      }
+      fileMinMaxMapListOfAllJoinColumns += ((fileMinMaxMap, joinColumn))
+    }
+  }
+
+  /**
+   * This method updates the min max map of the block if the value is less than min or more
+   * than max
+   */
+  private def updateMapIfRequiredBasedOnMinMax(fileMinMaxMap: mutable.Map[String, (AnyRef, AnyRef)],
```

Comment:
move param to next line, please follow the coding convension

Reply:
done